presto-orc/src/test/java/com/facebook/presto/orc/TestOrcWriter.java

```diff
@@ -120,4 +125,86 @@ public void testWriteOutputStreamsInOrder()
             }
         }
     }
+
+    @Test(expectedExceptions = IllegalStateException.class)
+    public void testVerifyIllegalStateException()
+            throws IOException
+    {
+        for (OrcWriteValidationMode validationMode : OrcWriteValidationMode.values()) {
+            TempFile tempFile = new TempFile();
```

Comment:
Remove the temp file. `MockOrcDataSink` can just be a blackhole only writing data to nowhere. Not necessary to be a concrete file.